src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4192,10 +4195,14 @@ def add_functionapp_vnet_integration(cmd, name, resource_group_name, vnet, subne
     functionapp = get_functionapp(cmd, resource_group_name, name)
     parsed_plan_id = parse_resource_id(functionapp.server_farm_id)
     plan_info = client.app_service_plans.get(parsed_plan_id['resource_group'], parsed_plan_id['name'])
+    app = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get', slot, client=client)
```

Comment:
I think you might be able to just use the `functionapp` variable instead of creating a new variable